src/main/java/seedu/address/ui/TravelPlanPanel.java

```diff
@@ -6,23 +6,30 @@
 import javafx.scene.control.Label;
 import javafx.scene.layout.Region;
 import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.Logic;
 import seedu.address.model.Directory;
 import seedu.address.model.travelplan.TravelPlan;
+import seedu.address.model.wishlist.Wishlist;
 
 
 /**
  * Panel containing basic information of a Travel Plan.
  */
 public class TravelPlanPanel extends UiPart<Region> {
     private static final String TravelPlanFXML = "TravelPlanPanel.fxml";
+    private static final String TOTAL_COST = "Total Cost: ";
     private final Logger logger = LogsCenter.getLogger(TravelPlanPanel.class);
 
     private Directory directory;
 
+    private Logic logic;
```

Comment:
Why do you need logic here tho? Doesn't seem to be used anywhere in the changes in this class?

Reply:
deleted